packages/webapp/src/containers/Finances/Report/reportFormattingUtils.js

```diff
@@ -73,5 +77,10 @@ export const createDefaultTypeFilter = ({ types, translate, typeCategory }) => {
     filterObject['LABOUR'] = { active: true, label: translate('SALE.FINANCES.LABOUR_LABEL') };
   }
 
+  // Sort the config object by label
+  filterObject = Object.fromEntries(
```

Comment:
NIT, but whenever possible it's preferable to use `const`, and in this case use a new name for your sorted object such as `sortedFilterObject`. That way when reading this we know for sure what each variable stores, and we know for sure that it won't be reassigned to something different further along.